ai/core/src/unitycatalog/ai/core/databricks.py

```diff
@@ -742,6 +745,26 @@ def from_dict(cls, config: Dict[str, Any]):
         accept_keys = ["profile"]
         return cls(**{k: v for k, v in config.items() if k in accept_keys})
 
+    def get_python_callable(self, function_name: str) -> str:
```

Comment:
The name is a bit confusing since I would expect a Callable, but it's actually a string 🤔

Reply:
100% agreed. I think `get_function_source` is a bit more expository. I've updated in #912 and I'm going to close this PR since the stacked nature of this is now out of sync and would be harder to fully review in a fragmented state.